app/services/metrics/taxonomy_count.rb

```diff
@@ -0,0 +1,13 @@
+module Metrics
+  class TaxonomyCount
+    def self.count_taggings_per_level
```

Comment:
From looking at the code, I think this is counting content, rather than tags/taggings.